stock_scanner_receipt/data/Receipt/scanner_scenario_step_receipt_done.py

```diff
@@ -0,0 +1,21 @@
+# flake8: noqa
```

Comment:
As explained on another PR, flake8 is disabled in these files, because some variables are only available at runtime (this code is evaluated during the scenario).